api/controllers/fileUploader.py

```diff
@@ -19,7 +20,7 @@
 @router.route('/image', methods=['POST'])
 def uploadImage():
     try:
-        data = request.get_json()
+        data = request.get_json()['imgFile']
```

Comment:
any specific reason you added `['imgFile']`? as image variable extracts `imgFile` so why you are adding it here again? Please remove it from either of the variables

Reply:
to verify that the data contains `imgFile` :/

Reply:
It complies with the ember data standard as I have mentioned in you PR please have a look at how ember data is functioning and as @gabru-md said it needs as the Application Adaptor is created for that particular model. 😀